app/js/controllers.js

```diff
@@ -3537,7 +3537,7 @@ angular.module('myApp.controllers', ['myApp.i18n'])
     }
 
     $scope.$on('history_delete', function (e, historyUpdate) {
-      if (historyUpdate.msgs[$scope.messageID]) {
+      if (historyUpdate && historyUpdate.msgs && historyUpdate.msgs[$scope.messageID]) {
```

Comment:
A safe getter like [lodash's one](https://lodash.com/docs/4.17.4#get) would have been handy here, but it is not a dependency, yet. Maybe ask @zhukov about it? It's possible to create tailored imports.